src/garage/envs/base.py

```diff
@@ -154,14 +154,16 @@ def __getstate__(self):
         # detect gym.Wrapper
         while issubclass(env.__class__, gym.Wrapper):
```

Comment:
AFAICT this statement should only ever execute once, because `env.unwrapped` is recursive: https://github.com/openai/gym/blob/master/gym/core.py#L253